src/sentry/static/sentry/app/views/performance/landing/content.tsx

```diff
@@ -114,45 +122,69 @@ class LandingContent extends React.Component<Props, State> {
     switch (display) {
       case LandingDisplayField.ALL:
         return this.renderLandingAll(summaryConditions);
-      case LandingDisplayField.FRONTEND:
-        return this.renderLandingFrontend(summaryConditions);
+      case LandingDisplayField.FRONTEND_PAGELOAD:
+        return this.renderLandingFrontend(summaryConditions, true);
+      case LandingDisplayField.FRONTEND_NAVIGATION:
+        return this.renderLandingFrontend(summaryConditions, false);
       case LandingDisplayField.BACKEND:
         return this.renderLandingBackend(summaryConditions);
       default:
         throw new Error(`Unknown display: ${display}`);
     }
   }
 
-  renderLandingFrontend = summaryConditions => {
+  renderLandingFrontend = (summaryConditions, isPageload) => {
```

Comment:
It looks to me that `pageload` differs enough from `navigation` that it may warrant two separate render methods for the two so we can customize them independently.

Reply:
Yeah, we can split them later though if that's the case, we might not leave it as 'navigation', but rather 'other' since there could be a 3rd op, but nav + another op wouldn't have vitals (same experience).